x-pack/plugins/security_solution/cypress/screens/timeline.ts

```diff
@@ -103,7 +105,7 @@ export const TIMELINE_FIELDS_BUTTON =
   '[data-test-subj="timeline"] [data-test-subj="show-field-browser"]';
 
 export const TIMELINE_FILTER = (filter: TimelineFilter) => {
-  return `[data-test-subj="filter filter-enabled filter-key-${filter.field} filter-value-${filter.value} filter-unpinned"]`;
+  return `[data-test-subj="filter filter-enabled filter-key-${filter.field} filter-value-${filter.value} filter-unpinned "]`;
```

Comment:
was this extra space on the end here meaningful in some way?

Reply:
As it is how it looks atm, and the component is not made by us. Update it here to fit current case and get the right element.

Reply:
Sounds good. Could you add a comment to that effect? Someone's editor may accidentally fix that on save or something and recreate the error. Might be worth contacting the code owners of the component too, to just remove the space in their code as well.

Reply:
I agree with @michaelolo24, we should fix the code and see if break anything else